org-jira.el

```diff
@@ -1517,6 +1527,27 @@ purpose of wiping an old subtree."
         (filename (org-jira-parse-issue-filename)))
     (org-jira-update-issue-details issue-id filename :assignee nil)))
 
+;;;###autoload
+(defun org-jira-report-issue ()
```

Comment:
I'm closely following the convention, (from `org-jira-assign-issue`), here but do have some concern that a function named `org-jira-report-issue` could be confused with `org-jira-create-issue`.

Listening for feedback on this one.

Reply:
Yes, I think this is confusing - org-jira-assign-issue is more part of a typical Jira workflow (send an issue to a person), while changing the reporter is usually never done (since it is auto-set based on who used Jira to file the issue).

Also report issue and create issue, as you mention, have the same connotation.

What about something like org-jira-change-issue-reporter ? (it would set a new precedent then, if we had any other edit one property at a time type modifications - where we could name them all org-jira-change-issue-xyz).

Reply:
Would `org-jira-set-issue-xyz` be preferable? 
The function might not change the value on each invocation, so "set" has the advantage of being a little more generic.

Reply:
works for me!